shells/pipes-shell/source/verbs/parse.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import {Utils} from '../../../lib/utils.js';
+import {logsFactory} from '../../../../build/runtime/log-factory.js';
+
+const {log} = logsFactory('pipe::parse');
+
+// can provide either a path or literal content for a manifest
+export const parse = async ({path, content}, tid, bus) => {
```

Comment:
this doesn't add the parsed manifest to the `context`, so if i call `runArc` later with one of the parsed recipes, it won't be available, or am i reading it wrong?

Reply:
I think we'll need to pass in the manifest path as well.

Reply:
@mmandlis correct, because I suspect the notion of selecting recipes out of the context is non-sensical for these use-cases. This is what I’ve been trying to say in chat.
The `run-arc` api is based on web-shell mechanics, that’s a different circumstance, and I want to avoid jumping through unnecessary hoops out of habit.  
As of now, there is no way to instantiate these recipes. This requires another change to the pipes-shell api that we should define. I didn’t want to implement that until we are all on the same page.

Reply:
today's Arcs for Android relies on the `Runtime` object (`runtime.ts`) that in turn relies on global context (gets a context in ctor), so in that sense the new `parse` API doesn't get us any closer to where we're trying to get. I'll schedule vc for us on Monday to discuss.

I'm fine with you merging this as an incremental step.

Reply:
+1 on merging this as an incremental step.

Reply:
I would say it does get us closer, and agree it is therefore a useful incremental step. It's also otherwise harmless so I'm merging it.